### PR TITLE
Melissa/onboarding review

### PR DESCRIPTION
**Pull request by me-liu**
[//]: # "Feel free to customize this template to your project's vibes"

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Added a review page that allows the user to review the information given previously

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 
[
Screenshot 2024-11-02 at 10 12 19 AM
](url)

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- Review all at once! No major changes made iteratively, mostly styling

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- I'm rly not sure if pushing to supabase works... need to troubleshoot more 

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @celinechoiii

### CODE REVIEW

File: app/onboarding/yay/page.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  ContinueText,
+  InlineContainer,
+  ReviewContainer,
+  Title,
+} from './styles';
+
+export default function Onboarding() {
+  return (
+    <Background>
+      <Circle />
```
Comment (celinechoiii) [inline app/onboarding/yay/page.tsx]: can you add the rose svg according to the figma?

File: app/onboarding/yay/page.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <InlineContainer>
+        <ReviewContainer>
+          <Title>You're all set!</Title>
+          <text>
+            Lorem ipsum dolor sit amet consectetur adipiscing elit, sed do
```
Comment (celinechoiii) [inline app/onboarding/yay/page.tsx]: replace the placeholder text with this: We recommend checking out some upcoming events in the Discover page. If you’re interested in participating, please sign up!

File: app/onboarding/yay/styles.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+export const InlineContainer = styled.main`
+  width: 25%;
+  flex-direction: column;
+  margin-top: 2%;
+  margin-bottom: 2%;
+  @media (max-width: 1200px) {
+    width: 30%;
+  }
+  @media (max-width: 768px) {
+    width: 55%;
```
Comment (celinechoiii) [inline app/onboarding/yay/styles.ts]: `width: 80%`

File: app/onboarding/yay/styles.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  }
+`;
+
+export const Circle = styled.main`
+  width: 85px;
+  height: 85px;
+  flex-shrink: 0;
+  background-color: #f7c1bd;
```
Comment (celinechoiii) [inline app/onboarding/yay/styles.ts]: use global colors

File: app/onboarding/yay/styles.ts [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+`;
+
+export const ContinueButton = styled.button`
+  display: flex;
+  padding: 12px 16px;
+  justify-content: center;
+  align-items: center;
+  align-self: stretch;
+  border-radius: 99999px;
+  background: var(--gray-12, #202020);
```
Comment (celinechoiii) [inline app/onboarding/yay/styles.ts]: global colors

File: app/onboarding/yay/styles.ts [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  border-style: solid;
+  border-color: ${COLORS.gray12};
+  cursor: pointer;
+`;
+
+export const ContinueText = styled.text`
```
Comment (celinechoiii) [inline app/onboarding/yay/styles.ts:63]: use `SMALL` from global styles. you probs don't need the `ContinueText` at all

File: app/onboarding/yay/page.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+            eiusmod tempor incididunt ut labore.
+          </text>
+          <ContinueButton>
+            <ContinueText>Continue</ContinueText>
```
Comment (celinechoiii) [inline app/onboarding/yay/page.tsx]: import `SMALL` and replace with `<SMALL>Continue<SMALL>`. might need to put in extra props such as colors but not entirely sure

File: app/onboarding/yay/styles.ts [hunk 8]
```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import styled from 'styled-components';
+import { Sans } from '@/styles/fonts';
+import COLORS from '../../../styles/colors';
```
Comment (celinechoiii) [inline app/onboarding/yay/styles.ts]: replace route with `@/styles/colors`

File: app/onboarding/review/styles.ts [hunk 9]
```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import NextImage from 'next/image';
+import styled from 'styled-components';
+import { Sans } from '@/styles/fonts';
+import COLORS from '../../../styles/colors';
```
Comment (celinechoiii) [inline app/onboarding/review/styles.ts]: `import COLORS from '@/styles/colors';`

File: app/onboarding/review/styles.ts [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  background: ${COLORS.gray12};
+  border-style: solid;
+  border-color: ${COLORS.gray12};
+  cursor: pointer;
+`;
+
+export const ConfirmText = styled.text`
```
Comment (celinechoiii) [inline app/onboarding/review/styles.ts:69]: no need for `ConfirmText`, use `SMALL` from global text

File: app/onboarding/review/styles.ts [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  background: #fefdfc;
+  border-radius: 16px;
+  box-shadow: 4px 4px 4px 0px rgba(0, 0, 0, 0.15);
+`;
+
+export const SmallText = styled.text`
+  color: #515151;
```
Comment (celinechoiii) [inline app/onboarding/review/styles.ts]: global colors

File: app/onboarding/review/styles.ts [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+  padding: 32px 32px 32px 32px;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  background: #fefdfc;
+  border-radius: 16px;
+  box-shadow: 4px 4px 4px 0px rgba(0, 0, 0, 0.15);
+`;
+
+export const SmallText = styled.text`
```
Comment (celinechoiii) [inline app/onboarding/review/styles.ts:36]: same thing here, try to use global text! this will also be `SMALL`

here's a good example: `<H6 $fontWeight="500">`
